src/main/java/com/jc/tm/service/TaskServiceImpl.java

```diff
@@ -0,0 +1,196 @@
+package com.jc.tm.service;
+
+import com.jc.tm.database.Status;
+import com.jc.tm.database.dao.CommentDao;
+import com.jc.tm.database.dao.TaskDao;
+import com.jc.tm.database.entity.Comment;
+import com.jc.tm.database.entity.Task;
+
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.util.Collection;
+
+public class TaskServiceImpl implements ITaskService {
+
+    private TaskDao taskDao;
+    private CommentDao commentDao;
+
+    public TaskServiceImpl(TaskDao taskDao, CommentDao commentDao) {
+        this.taskDao = taskDao;
+        this.commentDao = commentDao;
+    }
+
+    @Override
+    public Task saveTask(Task newTask) {
+        Task insetredTask = null;
+        try {
+             insetredTask = taskDao.insert(newTask);
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+        return insetredTask;
+    }
+
+    @Override
+    public Task removeTask(Long id) {
+        var task = getTask(id);
+        try {
+            taskDao.delete(task);
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+        return task;
+    }
+
+    @Override
+    public Task removeTask(Task task) {
+        try {
+            taskDao.delete(task);
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+        return task;
+    }
+
+    @Override
+    public Task updateTask(Task freshTask) {
+        try {
+            taskDao.update(freshTask);
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+        return freshTask;
+    }
+
+    @Override
+    public Task getTask(Long id) {
+        Task taskById = null;
+        try {
+            taskById = taskDao.getById(id);
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+        return taskById;
+    }
+
+    @Override
+    public Task getTask(Task task) {
+        try {
+            taskDao.getAll();
```

Comment:
wrong logic, this method should load data by id, the purposu of this methed it is refresh data